hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneClientAdapterImpl.java

```diff
@@ -190,14 +190,19 @@ public InputStream readFile(String key) throws IOException {
     }
   }
 
+  @Deprecated
   protected void incrementCounter(Statistic objectsRead) {
     //noop: Use OzoneClientAdapterImpl which supports statistics.
```

Comment:
My intention is to encourage new code to explicitly specify `count` since incrementing by 1 is now "old" with the introduction of batch delete / batch rename.

And all the existing calls have been migrated to the new one `incrementCounter(Statistic objectsRead, long count)`.

Actually I can just remove the old method since it is `protected`?

Reply:
Done. Removed deprecated annotation.